build broken on s390x after recent commit

Build on s390x is broken recent change to `tramp-s390x.c` in commit 9623156cfdfdfcc55230160942a45baff98083b8
````
make[4]: Entering directory '/home/jenkins/workspace/Mono/label/s390x/mono/mini'
  CC       libmini_la-tramp-s390x.lo
tramp-s390x.c: In function ‘mono_arch_get_unbox_trampoline’:
tramp-s390x.c:106:69: error: ‘m’ undeclared (first use in this function); did you mean ‘tm’?
  106 |  MonoMemoryManager *mem_manager = m_method_get_mem_manager (domain, m);
      |                                                                     ^
      |                                                                     tm
tramp-s390x.c:106:69: note: each undeclared identifier is reported only once for each function it appears in
...
````
seems the fix is to `s/m/method/`, I'll prepare a PR

CC: @vargaz @nealef

--- a/src/mono/mono/mini/tramp-s390x.c
+++ b/src/mono/mono/mini/tramp-s390x.c
@@ -98,7 +98,7 @@ typedef struct {
  */
 
 gpointer
-mono_arch_get_unbox_trampoline (MonoMethod *method, gpointer addr)
+mono_arch_get_unbox_trampoline (MonoMethod *m, gpointer addr)
 {
 	guint8 *code, *start;
 	int this_pos = s390_r2;
@@ -115,9 +115,9 @@ mono_arch_get_unbox_trampoline (MonoMethod *method, gpointer addr)
 	g_assert ((code - start) <= 28);
 
 	mono_arch_flush_icache (start, code - start);
-	MONO_PROFILER_RAISE (jit_code_buffer, (start, code - start, MONO_PROFILER_CODE_BUFFER_UNBOX_TRAMPOLINE, method));
+	MONO_PROFILER_RAISE (jit_code_buffer, (start, code - start, MONO_PROFILER_CODE_BUFFER_UNBOX_TRAMPOLINE, m));
 
-	snprintf(trampName, sizeof(trampName), "%s_unbox_trampoline", method->name);
+	snprintf(trampName, sizeof(trampName), "%s_unbox_trampoline", m->name);
 
 	mono_tramp_info_register (mono_tramp_info_create (trampName, start, code - start, NULL, NULL), domain);
 

